tests/test_api_handler.py

```diff
@@ -0,0 +1,29 @@
+import unittest
+import asyncio
+from aiounittest import async_test
+from Modules.api_handler import *
+
+
+class APIHandlerTest(unittest.TestCase):
+    handler : BaseWebsocketAPIHandler
+
+    def setUp(self):
+        handler = BaseWebsocketAPIHandler(hostname="dev.api.fuelrats.com", tls=True)
+
+    @async_test
+    async def test_connect(self):
+        if self.handler.connected: await self.handler.disconnect()
+        await self.handler.connect()
+        self.assertIsNotNone(self.handler.connected)
+
+    @async_test
+    async def test_disconnect(self):
+        self.assertIsNotNone(self.handler.connected)
+        await self.handler.disconnect()
+        self.assertIsNone(self.handler.connected)
```

Comment:
The `connected` property is a boolean, you'll want assertTrue and assertFalse here, respectively.